src/core/account/lobby-api.js

```diff
@@ -73,7 +73,7 @@ async function approveLoginRequest(
   }
 
   // Create a sanitized login stash object:
-  const stashTree = getStash(ai, username)
+  const { stashTree } = getStashById(ai, loginTree.loginId)
```

Comment:
What's the motivation for using `getStashById` over `getStash` here? The change incurs a BFS where there wasn't one before. Was this a bug?